backend/src/auth/config.rs

```diff
@@ -341,6 +341,10 @@ pub(crate) struct RoleConfig {
     #[config(default = "ROLE_TOBIRA_CAN_CREATE_SERIES")]
     pub(crate) can_create_series: String,
 
+    /// The role granting permission to create new playlists through Tobira.
+    #[config(default = "ROLE_TOBIRA_CAN_CREATE_PLAYLISTS")]
+    pub(crate) can_create_playlists: String,
```

Comment:
You reckon we want to set the default to `ROLE_USER`? :thinking: mhhh. I mean, this right now mirrors the role for series. But we also have `user_realm` default to `ROLE_USER` because we said all users should have that ability. "Power to the people" and all. And it does not come with any disadvantages as user realms are not findable via homepage. Same could apply to playlists, right? (But does the same also apply to series?)

I generally like to give normal users more power and by choosing the default, we can steer institutions a bit. I rather institutions have to disable this if they really want to, instead of the other way around. But yeah, I'm unsure here and wonder what you think.